src/System.Runtime.Extensions/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+MembersMustExist : Member 'System.IO.Path.EndsInDirectorySeparator(System.ReadOnlySpan<System.Char>)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.IO.Path.EndsInDirectorySeparator(System.String)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.IO.Path.TrimEndingDirectorySeparator(System.ReadOnlySpan<System.Char>)' does not exist in the implementation but it does exist in the contract.
+MembersMustExist : Member 'System.IO.Path.TrimEndingDirectorySeparator(System.String)' does not exist in the implementation but it does exist in the contract.
```

Comment:
tried with `ApiCompatBaseline.uapaot.txt` but ti doesn't compile local.

Reply:
What error? We shoukd use specific file if we can.
Btw I will be less available this coming week, @jeremykuhne can help

Reply:
If I add record to `ApiCompatBaseline.uapaot.txt I get:
```
...
C:\Users\marco_000\.nuget\packages\microsoft.dotnet.apicompat\1.0.0-beta.19177.11\build\Microsoft.DotNet.ApiCompat.targets(72,5): error : MembersMustExist : Member 'System.IO.Path.EndsInDirectorySeparator(System.ReadOnlySpan<System.Char> )' does not exist in the implementation but it does exist in the contract. [D:\git\corefx\src\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj]
C:\Users\marco_000\.nuget\packages\microsoft.dotnet.apicompat\1.0.0-beta.19177.11\build\Microsoft.DotNet.ApiCompat.targets(72,5): error : MembersMustExist : Member 'System.IO.Path.EndsInDirectorySeparator(System.String)' does not exist i n the implementation but it does exist in the contract. [D:\git\corefx\src\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj]
C:\Users\marco_000\.nuget\packages\microsoft.dotnet.apicompat\1.0.0-beta.19177.11\build\Microsoft.DotNet.ApiCompat.targets(72,5): error : MembersMustExist : Member 'System.IO.Path.TrimEndingDirectorySeparator(System.ReadOnlySpan<System.C har>)' does not exist in the implementation but it does exist in the contract. [D:\git\corefx\src\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj]
C:\Users\marco_000\.nuget\packages\microsoft.dotnet.apicompat\1.0.0-beta.19177.11\build\Microsoft.DotNet.ApiCompat.targets(72,5): error : MembersMustExist : Member 'System.IO.Path.TrimEndingDirectorySeparator(System.String)' does not exi st in the implementation but it does exist in the contract. [D:\git\corefx\src\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj]
C:\Users\marco_000\.nuget\packages\microsoft.dotnet.apicompat\1.0.0-beta.19177.11\build\Microsoft.DotNet.ApiCompat.targets(85,5): error : ApiCompat failed for 'D:\git\corefx\artifacts\bin\System.Runtime.Extensions\uap-Windows_NT-Debug\Sy stem.Runtime.Extensions.dll' [D:\git\corefx\src\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj]
```
With less specific `ApiCompatBaseline.txt` it works.

Reply:
@anipik

Reply:
@MarcoRossignoli is it failing for just uapaot or for other configurations  as well ?

Reply:
@Anipik thank's my fault messed up uap/uapaot added both now, one question for netfx is expected that we cannot build `src`
```
D:\git\corefx (exposetrimendingdirectoryseparator -> origin)
λ msbuild /v:m /t:Rebuild /p:TargetGroup=netfx src\System.Runtime.Extensions\src\
Microsoft (R) Build Engine version 15.9.21+g9802d43bc3 for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

C:\Program Files\dotnet\sdk\2.2.104\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.TargetFrameworkInference.targets(96,5): error NETSDK1013: The TargetFramework value '' was not recognized. It may be misspelled. If not, then the TargetFram eworkIdentifier and/or TargetFrameworkVersion properties must be specified explicitly. [D:\git\corefx\src\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj]

```